CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Versions
 * Use only major Node version to name node images
 * Add Node 12
 * Upgrade ScoutSuite to 5.3.3
+- Upgrade Golang version: 1.13
+- Upgrade Serverless version: 1.51
```

Comment:
I don't think you need to add this line since the Serverless image has not been released yet.
WDYT @mremi ?